utils/hct/gen_intrin_main.txt

```diff
@@ -411,6 +411,8 @@ namespace Texture1DMethods {
 // Use float for DXIL don't support f16 on CalcLOD.
 float [[ro]] CalculateLevelOfDetail(in sampler1d s, in float<1> x) : tex1d_t_calc_lod;
 float [[ro]] CalculateLevelOfDetailUnclamped(in sampler s, in float<1> x) : tex1d_t_calc_lod_unclamped;
+float [[ro]] CalculateLevelOfDetail(in sampler_cmp s, in float<1> x) : tex1d_t_calc_lod;
+float [[ro]] CalculateLevelOfDetailUnclamped(in sampler_cmp s, in float<1> x) : tex1d_t_calc_lod_unclamped;
```

Comment:
As a result of #5508, you can use `any_sampler` in the existing overloads without having to add new ones explicitly.